download/scala3.md

```diff
@@ -0,0 +1,7 @@
+---
+layout: downloadpage
+title: Download Scala 3
+
+release_version: 3.0.0
+release_date: "May 14, 2021"
+---
```

Comment:
Don’t you need to add the `license`?

Reply:
You should, yes. But then a lot of the other release notes also didn't so I followed the lead. Probably a bad example.

Reply:
Wait, I added it to the release notes in a later commit. I should also add it here, I guess.